### PR TITLE
Add more details in descriptions of OIDC schema

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

- add more details in description for OIDC username prefix and signing algorithms,
- mention that payload presented in the doc is in JSON format.

**Related issue(s)**

### CODE REVIEW

**Review by jaroslaw-pieszka — DISMISSED** [review]

File: components/kyma-environment-broker/internal/broker/plans_schema.go [hunk 1]
```diff
@@ -171,13 +171,13 @@ func NewOIDCSchema() *OIDCType {
...
 			UsernameClaim:  Type{Type: "string", Description: "The OpenID claim to use as the user name."},
-			UsernamePrefix: Type{Type: "string", Description: "If provided, all usernames will be prefixed with this value. If not provided, username claims other than 'email' are prefixed by the issuer URL to avoid clashes. To skip any prefixing, provide the value '-'."},
+			UsernamePrefix: Type{Type: "string", Description: "If provided, all usernames will be prefixed with this value. If not provided, username claims other than 'email' are prefixed by the issuer URL to avoid clashes. To skip any prefixing, provide the value '-' (dash character without additional characters)."},
 			SigningAlgs: Type{
 				Type: "array",
 				Items: &Type{
 					Type: "string",
 				},
-				Description: "List of allowed JOSE asymmetric signing algorithms.",
+				Description: "Comma separated list of allowed JOSE asymmetric signing algorithms, e.g. RS256,ES256",
```
Comment (IwonaLanger) [inline components/kyma-environment-broker/internal/broker/plans_schema.go:180]: Using the full form, `for example`, is better than the `e.g.` abbreviation in technical documentation. Please see the [Terminology table](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology) for more information.

Comment (szwedm) [reply components/kyma-environment-broker/internal/broker/plans_schema.go]: suggestion applied.

File: components/kyma-environment-broker/internal/broker/plans_schema.go [hunk 2]
```diff
@@ -171,13 +171,13 @@ func NewOIDCSchema() *OIDCType {
...
 			UsernameClaim:  Type{Type: "string", Description: "The OpenID claim to use as the user name."},
-			UsernamePrefix: Type{Type: "string", Description: "If provided, all usernames will be prefixed with this value. If not provided, username claims other than 'email' are prefixed by the issuer URL to avoid clashes. To skip any prefixing, provide the value '-'."},
+			UsernamePrefix: Type{Type: "string", Description: "If provided, all usernames will be prefixed with this value. If not provided, username claims other than 'email' are prefixed by the issuer URL to avoid clashes. To skip any prefixing, provide the value '-' (dash character without additional characters)."},
 			SigningAlgs: Type{
 				Type: "array",
 				Items: &Type{
 					Type: "string",
 				},
-				Description: "List of allowed JOSE asymmetric signing algorithms.",
+				Description: "Comma separated list of allowed JOSE asymmetric signing algorithms, for example RS256,ES256",
```
Comment (IwonaLanger) [inline components/kyma-environment-broker/internal/broker/plans_schema.go:180]:
```suggestion
				Description: "Comma separated list of allowed JOSE asymmetric signing algorithms, for example, RS256, ES256",
```

Comment (szwedm) [reply components/kyma-environment-broker/internal/broker/plans_schema.go]: suggestion applied.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**